src/client/components/Button/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Button.styles.css';
+
+export const Button = (props) => {
```

Comment:
Pass `onClick` through `props` to the `button` inside.

Reply:
implemented this can you check for correctness

Reply:
You should name it `onClick`, not `handleOnClick`. It is a convention that all event callbacks are prefixed with `on`.

Reply:
Yes true, changed